### PR TITLE
accountmanager: Set default gas limit.

### PR DESCRIPTION
**Pull request by j0sh**
Fixes a crash.

### CODE REVIEW

**Comment by j0sh** [comment]
Updated with a better solution. The crash happens when `-gasPrice` is not explicitly passed in. Here we simply use the default value from args, cast to `uint64`, rather than the unnecessary round-trip to/from `*big.Int`

**Review by yondonfu — COMMENTED** [review]
Should `client.Setup()` and `accountmanager.CreateTransactOpts` be updated to accept `uint64` instead of `big.Int` for the `gasLimit` parameter?

**Comment by j0sh** [comment]
Updating the parameters to explicitly be `uint64` seemed to fix the tests. I don't know how golang type inference works for function parameters.

File: cmd/livepeer/livepeer.go [hunk 1]
```diff
@@ -305,18 +305,12 @@ func main() {
...
-		if *gasLimit > 0 {
-			bigGasLimit = big.NewInt(int64(*gasLimit))
-		}
-
 		if *gasPrice > 0 {
 			bigGasPrice = big.NewInt(int64(*gasPrice))
 		}
 
-		err = client.Setup(*ethPassword, bigGasLimit, bigGasPrice)
+		err = client.Setup(*ethPassword, uint64(*gasLimit), bigGasPrice)
```
Comment (ericxtang) [inline cmd/livepeer/livepeer.go:313]: I think we can do `gasLimist.Uint64()`

**Review by yondonfu — APPROVED** [review]
LGTM! I don't think there are automatic type conversions of function parameters in Go - if we wanted to provide a value of an arbitrary type the function parameter type would have to be `interface{}`

File: cmd/livepeer/livepeer.go (hunk 1 above)
Comment (j0sh) [reply cmd/livepeer/livepeer.go:313]: `gasLimit` is an `*int` here (from the CLI args). Doesn't look like there's a `Uint64` method defined:

> gasLimit.Uint64 undefined (type *int has no field or method Uint64)

**Review by ericxtang — APPROVED** [review]

**Comment by j0sh** [comment]
My comment was more about the `gasLimit` type being omitted from the function signatures earlier; I don't know why golang inferred the param to be `*big.Int` which subsequently caused the tests to fail. All good now though!

**Pull request merged**